demo/arxiv_dataset/base.py

```diff
@@ -5,8 +5,10 @@
 from sqlalchemy import text
 
 
-def setup_and_register_schema_for_receiving_arxiv_data(engine):
-    db_name, schema_name = _setup_arxiv_schema(engine)
+def setup_and_register_schema_for_receiving_arxiv_data(
+    engine, schema_name='Arxiv'
```

Comment:
Gave schema_name a default value, so that the instructions for testing this remain the same.